src/services/NetworkService.ts

```diff
@@ -164,7 +178,7 @@ export class NetworkService {
         return isOffline
             ? new OfflineRepositoryFactory(networkType)
             : new RepositoryFactoryHttp(url, {
-                  websocketUrl: URLHelpers.httpToWsUrl(url) + '/ws',
+                  websocketUrl: url.indexOf('http:') !== -1 ? URLHelpers.httpToWsUrl(url) + '/ws' : URLHelpers.httpsToWsUrl(url) + '/wss',
```

Comment:
Do we really need this and also `httpsToWsUrl`?
It looks like `httpToWsUrl` already replaces `http` with `ws` thus if the protocol is "https" when it is replaced with "ws" it will become "ws**s**", no? And trailer `/wss` looks wrong.